constants.go

```diff
@@ -13,6 +13,13 @@ const (
 	TRACE   = "TRACE"
 )
 
+// Constants for PostForm
+const (
+	POSTFORM            = "POST"
```

Comment:
Will you please add  this constant in the list of HttpVerbs. Above along with GET POST, 

It will be great if you could add it after POST.

Reply:
I just deleted it since PostForm() isn't a verb/httpMethod exactly. The method is just a wrapper around post to indicate how the post is being sent.